lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -1199,4 +1385,17 @@ defmodule Trento.Domain.SapSystem do
         false
     end)
   end
+
+  defp update_instance(instances, instance_number, host_id, key, value) do
```

Comment:
I don't really like this `key/value` thing.
Could we send the whole new update and use maybe `Map.merge`?
https://hexdocs.pm/elixir/1.12/Map.html#merge/2